web/viewer.css

```diff
@@ -1084,7 +1093,6 @@ html[dir='rtl'] .verticalToolbarSeparator {
 }
 
 .toolbarLabel {
-  min-width: 16px;
```

Comment:
This does appear to be a better solution, and it reduces even more extraneous space than the former solution. I've pushed a commit with this change.